lib/parse.js

```diff
@@ -10,6 +10,7 @@ var defaults = {
     arrayLimit: 20,
     charset: 'utf-8',
     charsetSentinel: false,
+    collapseArrayIndices: true,
```

Comment:
it's better if boolean options default to `false`, so that absence and false are the same. can we rename this option so it can default to `false`?

Reply:
Renamed this option to `allowSparse`.